pkgs/top-level/all-packages.nix

```diff
@@ -17793,7 +17793,9 @@ with pkgs;
   coq_8_4 = coqPackages_8_4.coq;
   coq_8_5 = coqPackages_8_5.coq;
   coq_8_6 = coqPackages_8_6.coq;
-  coq_HEAD = callPackage ../applications/science/logic/coq/HEAD.nix {};
+  coq_HEAD = callPackage ../applications/science/logic/coq {
+    version = "8.7pre-2017-05-28";
```

Comment:
From https://github.com/coq/roadmaps/blob/master/text/roadmap-8.7.md, it is stated that the final version of Coq is scheduled at ~Oct 15 2017, and the current HEAD has yet to reach beta. As there is no tag yet for 8.7, I think the version should still be `8.6-unstable-2017-05-28`.